### DISCUSSION TITLE
What does this do?

### DISCUSSION THREAD

**Discussion by askpatrickw** [firstof9/ha-gasbuddy#77] [Q&A]
### Improve README

I know what Gas Buddy is, but the README doesn't explain exactly what functionality this adds to HA. For example:

- Does it just list one specific station?
- A list of hard-coded stations?
- Can you use the current location and list the top ones that match a filter you setup?

I use HA in my RV and my house so not sure where this would work best for me. 
If you provide notes, I'm happy to make a PR for the README.

**Comment by firstof9** [comment]
> Does it just list one specific station?

However many stations you'd like to monitor.

> A list of hard-coded stations?

No

> Can you use the current location and list the top ones that match a filter you setup?

As in via a device tracker if you're on the go? Not yet see #45
